### ISSUE TITLE
Make gravity generator do weird gravity stuff if emagged

### ISSUE THREAD

**Issue by doerakpoes** [yogstation13/yogstation#1431]
so bassicly i had a 5 second chat with john and thus come too the conclusion that the grav gen should do something weird when you emag it.

This is based on a 5 second chat i'm sure this will be a "great" idea.

**Comment by Ratvar** [comment]
Make it crush the server

**Comment by Kmc2000** [comment]
Make it attract any objects with mass towards each other.


Yes make the beef jerky come to ian instead

**Comment by Gintoki1** [comment]
I agree with KMC, and why isn't there more beef jerky on the station?

**Comment by doerakpoes** [comment]
So this is a bug now?
